README.md

```diff
@@ -185,31 +203,10 @@ Proxying multiple domains to container
 caddy.address=service1.example.com service2.example.com
 ```
 
-### Multiple caddyfile sections from one service/container
-It's possible to generate multiple caddyfile sections for the same service/container by suffixing the caddy prefix with _#. That's usefull to expose multiple service ports at different urls.
-
-Example:
-```
-caddy_0.address = portal.example.com
-caddy_0.targetport = 80
-caddy_1.address = admin.example.com
-caddy_1.targetport = 81
-```
-
-Generates:
-```
-portal.example.com {
-	proxy / servicename:80
-}
-admin.example.com {
-	proxy / servicename:81
-}
-```
-
 ### Docker configs
 You can also add raw text to your caddyfile using docker configs. Just add caddy label prefix to your configs and the whole config content will be prepended to the generated caddyfile.
 
-[Here is an example](examples/service-proxy.yaml#L4)
+[Here is an example](examples/example.yaml#L4)
 
```

Comment:
(This comment isn't really meant for this line)

I didn't see anything in the README about global options and how they're handled.

I think a global options label would be something like `caddy_0={{""}}` right?

What happens if two containers try to set the same options? I'm thinking maybe global options should only be allowed to be set on the caddy-docker-proxy container (maybe as labels, whatever you think is best) to avoid any potential conflicts.

Reply:
Everything in the README up to **Caddy label args creates a server block** are global options.
Global diretive:
```
caddy.tls = internal
```

Site-block directive:
```
caddy = localhost
caddy.tls = internal
```

Reply:
I will try to describe it better

Reply:
Hmm, that seems a bit confusing, I think maybe something like `caddy_global.admin = off` might feel better, instead of relying on the order in which labels are defined 🤔